adoc/Private-Registry-Requirements.adoc

```diff
@@ -164,8 +164,33 @@ Find out the URL of the database as well as the username and password that was u
 For Azure, the host name will likely look like `<postgres-server>.postgres.database.azure.com`.
 These will be required during the {spr} installation. You can also find it in the Azure Portal in the `Connection Strings` section.
 
-// TODO AI (Dirk Mueller) needs to be filled in
-// ==== AWS PostgreSQL
+==== AWS RDS
+
+{spr} is compatible with Amazon RDS using the Amazon Aurora-PostgreSQL or PostgreSQL service.
+
+The RDS database can be created from the AWS RDS Management Console or by using the `aws` cli using
+the following command:
+
+[source,bash]
+----
+aws rds create-db-instance
+    --engine postgres \
+    --allocated-storage 25 \
+    --db-instance-class db.t3.medium \
+    --db-security-groups <mydbsecuritygroup> \
+    --db-subnet-group <mydbsubnetgroup> \
+    --master-username <masterawsuser> \
+    --master-user-password <masteruserpassword> \
+    --backup-retention-period 3
+----
+
+When accessing the database from an on-premise {kube} cluster, set up the firewall rules accordingly.
+You can adapt Firewall rules from AWS Management Console as well.
+
+Please record the URL of the database as well as the username and password that was used when creating the database server.
+
+These will be required during the {spr} installation.
+
```

Comment:
We need to mention the need for creating some specific databases, just like we have in Amazon section.
We might even use the same text here:

_"After the server creation, it is necessary to manually create the following empty databases required: registry, notary_server and notary_signer."_

As there are probably no AWS specific commands for that, we might add something like:
_"Use your favorite client tool (like `psql`) to connect to the database server and create these empty databases."_